vision/.coveragerc

```diff
@@ -2,7 +2,7 @@
 branch = True
 
 [report]
-fail_under = 100
+# fail_under = 100
```

Comment:
Actually I might be able to remove that. I forgot about it.
All the hand-written code has full coverage; less certain about the GAPIC itself.